test/import/FSHImporter.Alias.test.ts

```diff
@@ -439,5 +439,28 @@ describe('FSHImporter', () => {
         /\$N\*tAll\*wed includes unsupported characters/
       );
     });
+
+    it('should resolve but log a warning when an alias contains ONLY a $', () => {
+      // Depending on how you read the FSH spec, this may technically be allowed FSH,
+      // but SUSHI will warn about it.
+      const input = leftAlign(`
+      Alias: $ = http://example.org
+
+      Profile: ObservationProfile
+      Parent: Observation
+      * code = $#foo
+      `);
+
+      const results = importText([new RawFSH(input, 'Alias.fsh')]);
+      expect(results.length).toBe(1);
+      const rule = results[0].profiles.get('ObservationProfile').rules[0] as AssignmentRule;
+      // The $ alias does still resolve because only a warning is logged
+      expect(rule.value).toEqual(
+        new FshCode('foo', 'http://example.org').withFile('Alias.fsh').withLocation([6, 10, 6, 14])
+      );
+      expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+      expect(loggerSpy.getAllMessages('warn')).toHaveLength(1);
+      expect(loggerSpy.getLastMessage('warn')).toMatch(/\$ includes unsupported characters/);
```

Comment:
Note: This warning message might be a little confusing for the case when the alias is just `$`. Because this case comes up so infrequently and when it does the alias name is referenced and the line number is provided, it didn't seem like it made sense to add another warning message into the code to complicate the implementation for this case. However, if either of you think we should have a more specific error message, I can probably be convinced.